autosklearn/ensemble_builder.py

```diff
@@ -811,7 +811,7 @@ def get_disk_consumption(self, pred_path):
     def score_ensemble_preds(self):
         """
             score predictions on ensemble building data set;
-            populates self.read_preds and self.read_scores
+            populates self.read_preds and self.read_losses
```

Comment:
Could you please rename this function to `compute_loss_per_model` or something that describes it better?